todocore/src/main/java/com/knighttodo/todocore/service/privatedb/repository/DayRepository.java

```diff
@@ -25,4 +26,7 @@ public interface DayRepository extends JpaRepository<Day, UUID> {
     @Modifying
     @Query("update DayTodo dt set dt.removed = true where dt.day.id=:dayId")
     void softDeleteAllDayTodosByDayId(@Param("dayId") UUID dayId);
+
+    @Query("select d from Day d where d.date = :date")
+    List<Day> findDaysByDate(@Param("date") Date date);
```

Comment:
why can't you use just findDaysByDate? without query? it should work.

Reply:
Done!